model/ftn/ww3_uprstr.ftn

```diff
@@ -934,17 +1406,404 @@
+!/    National Oceanic and Atmospheric Administration.  All rights
+!/    reserved.  WAVEWATCH III is a trademark of the NWS.
+!/    No unauthorized use without permission.
+!/
+!  1. Purpose :
+!     Updates the wind-sea part of the wave spectrum only 
+!  2. Method :
+!     
+!  3. Parameters :
+!
+!     Local parameters.
+!     ----------------------------------------------------------------
+!
+!  4. Subroutines used :
+!
+!     ----------------------------------------------------------------
+!  Internal Subroutines:
+!
+!  5. Called by :
+!
+!  6. Error messages :
+!
+!  7. Remarks :
+!
+!  8. Structure :
+!
+!  9. Switches :
+!
+!     !/T
+!
+! 10. Source code :
+!
+!/
+         REAL, DIMENSION(:), INTENT(INOUT)  :: VATMP
+         INTEGER, DIMENSION(:), INTENT(IN) :: VAMAPWS
+         REAL, INTENT(IN) :: PRCNTG
+         CHARACTER(25),PARAMETER :: myname='UPDTWSPEC'
+!
+!/T         WRITE (NDSO,*) trim(myname), ' starts'         
+         DO IK=1, NK
+            DO ITH=1, NTH
+               IF ( VAMAPWS(ITH+(IK-1)*NTH) .EQ. 1 ) THEN
+                   VATMP(ITH+(IK-1)*NTH) = VATMP(ITH+(IK-1)*NTH) * PRCNTG**2
+               END IF
+            END DO
+         END DO
+!
+!/T       WRITE (NDSO,*) trim(myname), ' ends'         
+      END SUBROUTINE UPDTWSPEC
+!/
+!/ ------------------------------------------------------------------- /
+!/ 
+      SUBROUTINE UPDTWSPECF(VATMP, PRCNTG, VAMAPWS, ISEA1p, ADJALL)  
+!/
+!/                  +-----------------------------------+
+!/                  | WAVEWATCH III           NOAA/NCEP |
+!/                  |         Andy Saulter              |
+!/                  |                        FORTRAN 90 |
+!/                  | Original code :       24-Oct-2018 |
+!/                  | Last update :         05-Oct-2019 |
+!/                  +-----------------------------------+
+!/
+!/    24-Oct-2018 : Original Code                       ( version 6.07 )
+!/
+!/    Copyright 2010 National Weather Service (NWS),
+!/    National Oceanic and Atmospheric Administration.  All rights
+!/    reserved.  WAVEWATCH III is a trademark of the NWS.
+!/    No unauthorized use without permission.
+!/
+!  1. Purpose :
+!     Updates the wind-sea part of the wave spectrum and shifts in frequency
+!     space 
+!  2. Method :
+!     
+!  3. Parameters :
+!
+!     Local parameters.
+!     ----------------------------------------------------------------
+!
+!  4. Subroutines used :
+!
+!     ----------------------------------------------------------------
+!  Internal Subroutines:
+!
+!  5. Called by :
+!
+!  6. Error messages :
+!
+!  7. Remarks :
+!
+!  8. Structure :
+!
+!  9. Switches :
+!
+!     !/T
+!
+! 10. Source code :
+!
+!/
+         REAL, DIMENSION(:), INTENT(INOUT)  :: VATMP
+         INTEGER, DIMENSION(:), INTENT(IN) :: VAMAPWS
+         REAL, INTENT(IN) :: PRCNTG
+         INTEGER, INTENT(IN) :: ISEA1p
+         LOGICAL, INTENT(IN) :: ADJALL
+         CHARACTER(25),PARAMETER :: myname='UPDTWSPEC'
```

Comment:
The name misses an F

Reply:
Thanks - will get this updated

Reply:
@flampouris - I've corrected the "myname" variable now. Thanks.